opdracht2.js

```diff
@@ -0,0 +1,106 @@
+//#### Opdracht 2 - Elementen in de DOM plaatsen
+//_Tip_: wanneer we meerdere waardes uit een array willen terugbrengen tot één getal of één string, gebruik je hier gewoon
+//een oude vertrouwde for-loop voor!
+
+//* **Opdracht 2a:** Hoeveel tv's zijn er al verkocht? Schrijf een script dat dit berekent. Log de uitkomst in de console.
+
+// 1. Declareer variabele
+// 2. Schrijf een for-loop die de hele inventory array heen door looped
+// 3. Bij elke iteratie kijken naar de verkochte tv's
+// 4. Sla deze tv's op in de variabele
+// 5. Log de uitkomst
+
+
+let totalSoldTvs = 0;
+for (let i = 0; i < inventory.length; i++) {
+    totalSoldTvs = totalSoldTvs + inventory[i].sold;
```

Comment:
Dit zou je iets korter kunnen schrijven:
```suggestion
    totalSoldTvs += inventory[i].sold;
```